x-pack/plugins/security_solution/common/api/endpoint/suggestions/get_suggestions.schema.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.0.0
+info:
+  title: Get Suggestions Schema
+  version: '2023-10-31'
+paths:
+  /api/endpoint/suggestions/{suggestion_type}:
+    post:
+      summary: Get suggestions
+      operationId: GetEndpointSuggestions
+      x-codegen-enabled: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - parameters
+              properties:
+                field:
+                  type: string
+                query:
+                  type: string
+                filters: {}
+                fieldMeta: {}
+      parameters:
+        - name: query
+          in: path
+          required: true
+          schema:
+            type: object
+            properties:
+              id:
```

Comment:
should this be `suggestion_type` instead?

Reply:
yes! thanks :)